sdk/python/symbolchain/ripemd160.py

```diff
@@ -1,11 +1,15 @@
 import hashlib
 
-if 'ripemd160' not in hashlib.algorithms_available:
-	from ripemd import ripemd160 as ripemd160_impl
+from ripemd import ripemd160 as ripemd160_impl
 
 
 def _factory():
-	return hashlib.new('ripemd160') if 'ripemd160' in hashlib.algorithms_available else ripemd160_impl.new()
+	try:
+		hasher = hashlib.new('ripemd160')
+	except ValueError:
+		hasher = ripemd160_impl.new()
+
+	return hasher
 
```

Comment:
this seems unrelated, also seems like bug in your py version. ^^

Reply:
this goes back to this discussion:
https://discord.com/channels/856325968096133191/925409482220118016/1023968691290918972

seems there is no way to check it otherwise